crates/fj-core/src/services/mod.rs

```diff
@@ -42,6 +42,18 @@ impl Services {
         }
     }
 
+    /// Construct an instance of `Services` with a pre-defined configuration for the validation service
+    pub fn with_validation_config(config: &ValidationConfig) -> Self {
```

Comment:
Seems a bit misleading to take a `&ValidationConfig` that is then copied inside. It would be better to directly take `ValidationConfig`.

But this is not a blocker. Happy to merge as-is!

Reply:
This has been addressed.